### DISCUSSION TITLE
does the logs support sanitazion?

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#113] [General]
am trying to make a feature request but there is some leakage in the logs. is there any way to sanitize it?

**Comment by euzu** [comment]
This is already in the main branch   but there is  
no release.

You need to do this manually with sed or another tool

**Reply by aniel300** [reply]
so it has been implemented but is not pushed the docker image?

**Reply by euzu** [reply]
yes you are right.

**Reply by aniel300** [reply]
ok i can sanitize it for the time being
